python/tk-core/python/tank/bootstrap/resolver.py

```diff
@@ -206,31 +206,30 @@ def resolve_not_found_sg_configuration(self, config_descriptor, sg_connection):
         # Validate if descriptor version token is omitted
         resolve_latest = is_descriptor_version_missing(config_descriptor)
         if (
-                constants.SGTK_CONFIG_LOCK_VERSION in os.environ
-                and sys.version_info[0] != 3
+                sys.version_info[0] < 3
```

Comment:
`and` missing here

Reply:
It is for the "unified view" of Github, here I show a split view:
![Screen Shot 2023-01-12 at 2 24 54 PM](https://user-images.githubusercontent.com/42726133/212162378-98bdb2f3-4a9c-4e03-b3ac-b608361ab036.png)

Reply:
Yep, I missed that. Sorry, you're right.